src/frontend/components/UI/QuestsViewer/components/QuestDetailsWrapper/index.tsx

```diff
@@ -380,10 +389,17 @@ export function QuestDetailsWrapper({
     questMeta !== undefined &&
     questRewards !== undefined
   ) {
+    const isRewardTypeClaimable = Boolean(
+      questMeta?.rewards?.some(
+        (reward) => rewardTypeClaimEnabled[reward.reward_type]
+      )
+    )
+
     const ctaDisabled =
       !flags.questsOverlayClaimCtaEnabled ||
       (!isEligible() && !showResyncButton) ||
-      isClaiming
+      isClaiming ||
+      !isRewardTypeClaimable
```

Comment:
we'll have to keep this in mind when we add multiple rewards per quest. that the cta button for claim all will be enabled even if we only want claim to be enabled for 1 of the n reward types for that quest

might be worth adding a TODO here for that